editoast/fga/src/client.rs

```diff
@@ -303,10 +311,13 @@ impl Client {
 
     /// Prepares multiple write requests to OpenFGA
     ///
-    /// OpenFGA Writes API do not accept more than 100 tuples per request.
+    /// OpenFGA Writes API has a maximum number of tuples it accepts per request
+    /// (default value: 100).
+    ///
     /// The [PreparedWrites] type returned by this function accepts any number
     /// of tuples through [PreparedWrites::push] and will chunk them into
-    /// requests of 100 tuples each. The requests are sent concurrently when
+    /// requests of n tuples each, with n the maximum number of tuple reads set
+    /// in the client settings. The requests are sent concurrently when
```

Comment:
```suggestion
    /// requests of n tuples each, with n the maximum number of tuple writes
    /// configured in [`ConnectionSettings::max_tuple_writes`]. The requests are sent concurrently when
```

better link the symbol for more precision, and using the brackets will raise a linting error if the symbol moves

Reply:
`[ConnectionSettings::limits::max_checks_per_batch_check]` doesn't seem to be a valid syntax so I went a bit verbose in the documentation. If you want it to be more concise I can remove either the `[ConnectionSettings::limits]` parts or the `[Limits::max_checks_per_batch_check]` parts.

Reply:
I misplaced the backticks :/ What you did is fine thanks